### PR TITLE
[CHANGE] online and offline counter

### PR DESCRIPTION
**Pull request by ClaudiaMia**
Fixes and changes online counter that did not work due to last PR.

Also highlights and counts offline characters in the room search list differenty.

### CODE REVIEW

File: pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx [hunk 1]
```diff
@@ -170,7 +170,21 @@ function RoomEntry({ roomInfo }: {
...
+					{ ' ' }
+					{ onlineCharacters }
+					{ ' ' }
+					<span className='offlineCount'>(+{ totalCharacters - onlineCharacters })</span>
+					{ ' ' }
+					/
+					{ ' ' }
+					{ maxUsers }
+					{ ' ' }
+					)
```
Comment (Sekkmer) [inline pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx:186]: Not sure if this is more readable or even work but I got this idea:
(feel free to close this without change)

```suggestion
					{ `${name} ( ${onlineCharacters} ` }
					<span className='offlineCount'>(+{ totalCharacters - onlineCharacters })</span>
					{ ` / ${maxUsers} )` }
```

**Review by Sekkmer — CHANGES_REQUESTED** [review]

File: pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ const TIPS: readonly string[] = [
 	`You can use your browser's "back" and "forward" buttons to navigate between screens.`,
 	`The dragging points to move room devices are invisible, but generally under the item.`,
 	`In the Pandora settings, character (chat) and account (direct messages) name colors are set separately.`,
+	`Every single change in the wardrobe happens instantly and is visible to everyone in the room immediately.`,
```
[the remark this answers is not in the document]
Comment (ClaudiaMia) [reply pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx:30]: Both is correct and sounds fine to me, so I will switch to your preference. :)

File: pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx (hunk 1 above)
Comment (ClaudiaMia) [reply pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx:186]: Yes, this seems to work, too. It is Jomshir's code, so I will ask if that change is okay. It surely looks more readable to me.

Comment (Jomshir98) [reply pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx]: The change looks good to me~

**Review by Sekkmer — APPROVED** [review]

**Pull request merged**